### PR TITLE
Audio parties

### PR DESCRIPTION
**Pull request by joextodd**
Simple group play at /party/beas

### CODE REVIEW

**Comment by joextodd** [comment]
Just needs uglify fixing with firebase

**Comment by lukejacksonn** [comment]
This looks good! Does it work as expected?

**Comment by joextodd** [comment]
Yep, very simple starting point. But changes tracks for anyone in the same party. I'll get it merged then. 

Also update on the streaming scenario (not giving in). Can't get webm to work on iPhone as they don't support it. The reason the mp4 streams aren't working is because the metadata is placed at the end of the file, and audio players won't start playing until that is received. So last attempt will be to hack the metadata on to the start otherwise I'm all out of ideas.

**Comment by lukejacksonn** [comment]
Cool, what about mp3 then?

File: package.json [hunk 1]
```diff
@@ -38,11 +38,12 @@
     "rollup-plugin-buble": "^0.15.0",
     "rollup-plugin-commonjs": "^8.0.2",
     "rollup-plugin-node-resolve": "^2.0.0",
-    "rollup-plugin-uglify": "^1.0.1",
-    "rollup-plugin-postcss": "^0.5.3"
+    "rollup-plugin-postcss": "^0.5.3",
+    "rollup-plugin-uglify-es": "0.0.1"
```
Comment (lukejacksonn) [inline package.json:42]: What is this for? Was it the compressing of firebase?

**Pull request merged**